tools/checkpoint_conversion/convert_vit_checkpoints.py

```diff
@@ -232,9 +232,8 @@ def convert_image_converter(hf_image_processor):
     mean = config["image_mean"]
     return ViTImageConverter(
         image_size=image_size,
-        scale=config["rescale_factor"],
-        norm_mean=mean,
-        norm_std=std,
+        scale=[config["rescale_factor"] / s for s in mean],
```

Comment:
`scale=[config["rescale_factor"] / s for s in std]`

uploaded presets values doesn't change since mean and std are same for Vit.